.github/workflows/bench.yml

```diff
@@ -74,7 +109,6 @@ jobs:
       env:
         RUN_CMD: cabal build benchmarks
       run: |
-        cabal update
```

Comment:
Why would we not want to do this?

Reply:
It's done as part of the "Calculate cabal cache key" step now.